lib/http-context.js

```diff
@@ -310,10 +314,15 @@ HttpContext.prototype.done = function () {
   if (this.req.query._format) {
     accepts = this.req.query._format.toLowerCase();
   }
-  var dataExists = typeof data !== 'undefined';
 
+  var acceptableMIMEType = accepts;
+  if (acceptableMIMEType) {
+    acceptableMIMEType = acceptableMIMEType.replace(/;\s*charset=[^;]+;*/i, '');
```

Comment:
@shelbys There might be other legitimate parameters, like `level=1`. Do we want to throw that away too?